jupyterlab_polus_render/pyproject.toml

```diff
@@ -1,18 +1,120 @@
-[tool.poetry]
+[build-system]
+requires = [
+    "hatchling>=1.21.1",
+    "jupyterlab>=4.0.0,<5",
+    "hatch-nodejs-version>=0.3.2",
+]
+build-backend = "hatchling.build"
+
+[project]
 name = "jupyterlab_polus_render"
-version = "0.1.1"
-description = "Displays Polus Render dashboard in Jupyter Notebooks."
-authors = ["Jeff Chen <Jeff.Chen@axleinfo.com>"]
-packages = [{ include = "polus"}, {include = "render-server-ext"}]
-License = "MIT"
+description = "A Jupyterlab extension for RenderUI."
 readme = "README.md"
-repository = "https://github.com/PolusAI/jupyterlab-extensions"
+license = { file = "LICENSE.txt" }
+requires-python = ">=3.9"
+authors = [
+    { name = "Akshat Saini, Jeff Chen", email = "akshat.saini@axleinfo.com" },
+]
+keywords = [
+    "IPython",
+    "Jupyter",
+    "Widgets",
+]
+classifiers = [
+    "Framework :: Jupyter",
+    "Intended Audience :: Developers",
+    "Intended Audience :: Science/Research",
+    "License :: OSI Approved :: BSD License",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.7",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12",
+]
+dependencies = [
+    "ipywidgets>=8.0.0",
+]
+version = "1.0.0"
 
-[tool.poetry.dependencies]
-python = ">=3.9" 
-render-server-ext = { path = "./render-server-ext/", develop = false }
+[project.optional-dependencies]
+docs = [
+    "jupyter_sphinx",
+    "nbsphinx",
+    "nbsphinx-link",
+    "pypandoc",
+    "pytest_check_links",
+    "recommonmark",
+    "sphinx>=1.5",
+    "sphinx_rtd_theme",
+]
+examples = []
+test = [
+    "nbval",
+    "pytest-cov",
+    "pytest>=6.0",
+]
 
+[project.urls]
+Homepage = "https://github.com/PolusAI/jupyterlab_polus_render"
```

Comment:
The correct URL is https://github.com/PolusAI/jupyterlab-extensions/tree/master/jupyterlab_polus_render